RestaurantReservationSystem/Seeders/TableSeeder.cs

```diff
@@ -0,0 +1,25 @@
+﻿using RestaurantReservation.Db.Models;
+
+namespace RestaurantReservation.Db.Seeders;
+
+public class TableSeeder
+{
```

Comment:
Also, we are exposing the DB context outside the DB project, all seeders should belong to the DB layer not the Domain/App layer. We won't to make sure we make the DB layer only about implementing all stuff related to DB and it's operations, and not expose db context to other projects. We just need to expose the repos, if someone wants anything from our DB, they should be talking to our repos and not access out DB directly.